Resources/doc/reference/configuration/headers.rst

```diff
@@ -215,7 +215,7 @@ directives are flags that are included when set to true:
 ``etag``
 """"""""
 
-**type**: ``boolean``
+**type**: ``enum`` **options**: ``true``, ``false``, ``strong``, ``weak``
```

Comment:
we need to remove `true` here as its not an option anymore (it still works, but we want people to use explicit options.

lets use quotes `"` around the string options to indicate that false is boolean and not the string boolean.